Description: _cmake_casing
 Pass package names in consistent casing to CMake directives, as CMake 3.18 has
 started complaining - needlessy - otherwise.
 .
 diff --git a/cmake/FindCUnit.cmake b/cmake/FindCUnit.cmake
 index 2c57048..ea108cb 100644
Author: Chris Hofstaedtler <zeha@debian.org>
Index: wslay/cmake/FindCUnit.cmake
===================================================================
--- wslay.orig/cmake/FindCUnit.cmake
+++ wslay/cmake/FindCUnit.cmake
@@ -12,7 +12,7 @@ find_library(CUNIT_LIBRARY NAMES cunit l
 # handle the QUIETLY and REQUIRED arguments and set CUNIT_FOUND to TRUE if
 # all listed variables are TRUE
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(CUNIT DEFAULT_MSG CUNIT_LIBRARY CUNIT_INCLUDE_DIRS)
+find_package_handle_standard_args(CUnit DEFAULT_MSG CUNIT_LIBRARY CUNIT_INCLUDE_DIRS)
 
 if(CUNIT_FOUND)
 	set(CUNIT_LIBRARIES ${CUNIT_LIBRARY})
Index: wslay/cmake/FindNettle.cmake
===================================================================
--- wslay.orig/cmake/FindNettle.cmake
+++ wslay/cmake/FindNettle.cmake
@@ -12,7 +12,7 @@ find_library(NETTLE_LIBRARY NAMES nettle
 # handle the QUIETLY and REQUIRED arguments and set NETTLE_FOUND to TRUE if
 # all listed variables are TRUE
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(NETTLE DEFAULT_MSG NETTLE_LIBRARY NETTLE_INCLUDE_DIRS)
+find_package_handle_standard_args(Nettle DEFAULT_MSG NETTLE_LIBRARY NETTLE_INCLUDE_DIRS)
 
 if(NETTLE_FOUND)
 	set(NETTLE_LIBRARIES ${NETTLE_LIBRARY})
